test/helpers.ts

```diff
@@ -77,3 +77,7 @@ export function cumsum(x: number[]): number[] {
     })
     return ret;
 }
+
+export function rep(x: any, n: number): (typeof x)[] {
```

Comment:
`rep` stands for 'repeat'?

Reply:
yeah, stolen from the R utility function of the same name (as with seq for sequence)